code/modules/research/designs/uncommented.dm

```diff
@@ -0,0 +1,69 @@
+
```

Comment:
Every datum in here is missing a sort strings. Not an issue unless they're uncommented, but the instructions are literally to just uncomment if the conditions are met, soooo...